pkg/config/config_loader.go

```diff
@@ -370,6 +370,11 @@ func getType(typeDef typeDefinition) (Type, error) {
 		}, nil
 
 	case typeDef.isClassic():
+
+		if typeDef.Api == ApiTypeBucket {
```

Comment:
@Laubi when creating a followup for not allowing the `api: format` please link to this line and what refactorings would be needed to split loading a bucket definition from a shorthand API definition

Reply:
Done